UPGRADE-7.2.md

```diff
@@ -112,6 +112,13 @@ TwigBridge
 
  * Deprecate passing a tag to the constructor of `FormThemeNode`
 
+TypeInfo
+--------
+
+ * Rename `Type::isA()` to `Type::isIdentifiedBy()` and `Type::is()` to `Type::isSatisfiedBy()`
+ * Remove `Type::getBaseType()`, `Type::asNonNullable()` and `Type::__call()` methods
```

Comment:
Can we maybe add relevant replacements for `getBaseType()` and `asNonNullable()`?

I personally couldn't find it quickly when checking out the pull request, which means it's probably even harder for users to find a way to fix their code.

Reply:
ping @mtarld

Reply:
Yes, you're right, we could write something like that instead?
```
* Remove `Type::getBaseType()`, use `WrappingTypeInterface::getWrappedType()` instead
* Remove `Type::asNonNullable()`, use `NullableType::getWrappedType()` instead
```

Reply:
That works for me.

Reply:
👍 updated